packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,160 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Implementation detail nuances:](#implementation-detail-nuances)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop               | Type                                                                     | Default     | Description                                                                                                                                              | Required |
+| ------------------ | ------------------------------------------------------------------------ | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content            | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                                   | ✅       |
```

Comment:
Should this be `title`? that is what we follow in other components. One difference is in other components we also have subtitle / description and here we don't but should we do it for consistency?

Anyway in HTML also that tooltip-like component itself is called `<a title="something">` so title doesn't seem to out of the way 🤔 ?

Reply:
but popover will be a different component right? how about calling it as `description` inline with our other components strategy? because it's not really a title and if we give title then its implicit that it might be bold or something?

Reply:
But they will have the same API just that popover will behave a bit differently, eg: click to open and no auto dismiss. 

eg: 
```jsx
<Popover content={<p>some jsx</p>}>
  <Button>Trigger</Button>
</Popover>
```